detox/test/e2e/config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   "rootDir": "../..",
   "testEnvironment": "./test/e2e/environment.js",
   "testRunner": "./test/node_modules/jest-circus/runner",
-  "setupFilesAfterEnv": ['./test/e2e/init-coverage.js'],
+  "setupFilesAfterEnv": [],
```

Comment:
Not really related, right?

Reply:
Yeah, but I've noticed that this file is redundant already. It should have gone with my other Coveralls fixes, but I missed it.